src/components/RadioButtonList/RadioButtonList.tsx

```diff
@@ -90,27 +108,67 @@ const RadioButtonList = <T extends RadioButtonOption>({
     })
   }
 
-  const renderRadioButton = (option: RadioButtonOption, index: number) => {
+  const renderRadioButton = (
+    option: RadioButtonOption,
+    index: number,
+    formValidation: RadioButtonListProps['formValidation']
+  ) => {
     let component = (
-      <StyledLabel
-        key={option.key || option.name}
-        htmlFor={option.key || option.name}
-        label={option.value}
-      >
-        <RadioButton
-          data-testid={`${groupName}-${option.key || option.name}`}
-          checked={option.key === defaultValue}
-          id={option.key || (option.name as string)}
-          onChange={() => {
-            if (onChange) {
-              // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-              // @ts-ignore
-              onChange(groupName, option)
-            }
-          }}
-          value={option.key}
-        />
-      </StyledLabel>
+      <Fragment key={option.key || option.name}>
+        <StyledLabel
+          key={option.key || option.name}
+          htmlFor={option.key || option.name}
+          label={option.value}
+        >
+          <RadioButton
+            data-testid={`${groupName}-${option.key || option.name}`}
+            checked={option.key === defaultValue}
+            id={option.key || (option.name as string)}
+            onChange={() => {
+              if (onChange) {
+                // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                // @ts-ignore
+                onChange(groupName, option)
+              }
+            }}
+            value={option.key}
+          />
+        </StyledLabel>
+
+        {option.open_answer &&
+          formValidation &&
+          formValidation.selectedRadioButton === option.value && (
+            <FormField
+              meta={{
+                label: ``,
+                name: option.name,
+                subtitle: '',
+              }}
+              hasError={(errorType) =>
+                formValidation.errors[`open_answer-${option.value}`]?.type ===
+                errorType
+              }
+              getError={(errorType) =>
+                formValidation.errors[`open_answer-${option.value}`]?.type ===
+                errorType
+              }
+            >
+              <TextArea
+                {...formValidation.register(`open_answer-${option.value}`)}
```

Comment:
Is the set value necessary and the trigger, I guess register already adds there through descructuring.

Reply:
I see that onchange does a thing under the hood, its checks the max content and shows chars left do never mind